### PR TITLE
docs(pull-request-template): add a pull request template

### LINKED ISSUE #58: Create a pull request template
Create a pull request template
an example can be found at https://github.com/americanexpress/one-app/blob/main/.github/pull_request_template.md

This tutorial can be followed

https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository

### PR DESCRIPTION
**Pull request by KandyKad**
Fixes #58 

Added a Pull Request Template

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] There aren't any other open Pull Requests for the same issue/update.
+- [ ] These changes should be applied to a maintenance branch.
+- [ ] This change requires cross browser checks.
+- [ ] Performance tests should be ran against the server prior to merging.
+- [ ] This change impacts caching for client browsers.
+- [ ] This change impacts HTTP headers.
+- [ ] This change adds additional environment variable requirements for One App users.
+- [ ] I have added the Apache 2.0 license header to any new files created.
+
+## What is the Impact to Developers Using One App?
```
Comment (JamesSingleton) [inline .github/pull_request_template.md:37]: This would be `holocron` instead of `One App`

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
+- [ ] My changes are in sync with the code style of this project.
+- [ ] There aren't any other open Pull Requests for the same issue/update.
+- [ ] These changes should be applied to a maintenance branch.
+- [ ] This change requires cross browser checks.
+- [ ] Performance tests should be ran against the server prior to merging.
+- [ ] This change impacts caching for client browsers.
```
Comment (JamesSingleton) [inline .github/pull_request_template.md:32]: There shouldn't be any changes within `holocron` that would impact client browsers

File: .github/pull_request_template.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] This change impacts HTTP headers.
```
Comment (JamesSingleton) [inline .github/pull_request_template.md:33]: There shouldn't be any changes within `holocron` to impact HTTP headers

File: .github/pull_request_template.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
+- [ ] My changes are in sync with the code style of this project.
+- [ ] There aren't any other open Pull Requests for the same issue/update.
+- [ ] These changes should be applied to a maintenance branch.
+- [ ] This change requires cross browser checks.
+- [ ] Performance tests should be ran against the server prior to merging.
```
Comment (JamesSingleton) [inline .github/pull_request_template.md:31]: There isn't a server for any performance tests to be ran against, this can be removed.

File: .github/pull_request_template.md [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Dependency update
+
+## Checklist:
+<!--- Go over all the following points, and put an `x` in all the boxes that apply. -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
+- [ ] My changes are in sync with the code style of this project.
+- [ ] There aren't any other open Pull Requests for the same issue/update.
+- [ ] These changes should be applied to a maintenance branch.
+- [ ] This change requires cross browser checks.
```
Comment (JamesSingleton) [inline .github/pull_request_template.md:30]: There shouldn't be any changes that impact browsers that there should need to be cross browser checks

**Review by JamesSingleton — CHANGES_REQUESTED** [review]

File: .github/pull_request_template.md [hunk 6]
```diff
@@ -0,0 +1,38 @@
...
+- [ ] Performance tests should be ran against the server prior to merging.
+- [ ] This change impacts caching for client browsers.
+- [ ] This change impacts HTTP headers.
+- [ ] This change adds additional environment variable requirements for One App users.
```
Comment (JAdshead) [inline .github/pull_request_template.md:34]: Why is this here ?

**Review by JAdshead — CHANGES_REQUESTED** [review]
Thank you for your contribution, please could you tailor the questions to the context of the packages this includes

File: .github/pull_request_template.md (hunk 1 above)
Comment (KandyKad) [reply .github/pull_request_template.md:37]: Ok.. I'll make the changes and update the document.

**Review by nellyk — APPROVED** [review]

**Comment by nellyk** [comment]
@KandyKad This looks great, thank you for contribution

**Review by Francois-Esquire — APPROVED** [review]

**Review by JAdshead — APPROVED** [review]

**Review by JamesSingleton — APPROVED** [review]

**Comment by nellyk** [comment]
@KandyKad Thank you so much for your contribution 🥳

**Comment by KandyKad** [comment]
Thank you for giving me an opportunity to contribute!!
